File: hackerrank/30daysOfCode/day1.js
/**
 * Javascript (Node.js) solution for {@link https://www.hackerrank.com/challenges/30-data-types/problem Day 1: Data Types}
 * @author Emelie Widegren <emeliewide@gmail.com>
 */

process.stdin.resume()
process.stdin.setEncoding('ascii')

var inputStdin = ''
var inputStdinArray = ''
var inputCurrentline = 0

process.stdin.on('data', function (data) {
  inputStdin += data
})

process.stdin.on('end', function () {
  inputStdinArray = inputStdin.split('\n')
  main()
})

// Reads complete line from STDIN
function readLine () {
  return inputStdinArray[inputCurrentline++]
}

function main () {
  var i = 4
  var d = 4.0
  var s = 'HackerRank '

  // Declare second integer, double, and String variables.
  // Read and save an integer, double, and String to your variables.
  var secondInteger = parseInt(readLine())
  var secondDouble = parseFloat(readLine())
  var secondStr = readLine()

  // Print the sum of both integer variables on a new line.
  console.log(i + secondInteger)

  // Print the sum of the double variables on a new line.
  console.log((d + secondDouble).toFixed(1))

  // Concatenate and print the String variables on a new line
  // The 's' variable above should be printed first.
  console.log(s + secondStr)
}

File: advent-of-code/2018/day1.py
#!/usr/bin/env python
"""
Python solution for Project Euler Problem 1: Multiples of 3 and 5
"""
__author__ = "Emelie Widegren"
__email__ = "emeliewide@gmail.com"


def multiples():
    sum = 0
    for i in range(1000):
        if (i % 3) == 0 or (i % 5) == 0:
            sum += i
    return sum


if __name__ == '__main__':
    print(multiples())
